src/test/osd/Object.h

```diff
@@ -517,9 +520,11 @@ class ObjectDesc {
 
   // takes ownership of gen
   void update(ContentsGenerator *gen, const ContDesc &next);
-  bool check(bufferlist &to_check);
+  bool check(bufferlist &to_check,
+	     const std::pair<uint64_t, uint64_t>& offlen);
   bool check_sparse(const std::map<uint64_t, uint64_t>& extends,
-		    bufferlist &to_check);
+		    bufferlist &to_check,
+		    const std::pair<uint64_t, uint64_t>& offlen);
```

Comment:
I'd suggest just passing the two u64 (as a pair) by value. Should be faster than a pointer to pair.

Reply:
Oops... it's just a test file...